sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/JsonSerializable.java

```diff
@@ -159,6 +161,15 @@ public void populatePropertyBag() {
         return getMapper().convertValue(this.propertyBag, HashMap.class);
     }
 
+    protected <T> Map<String, T> getMap(String propertyKey) {
```

Comment:
I think as per central SDK guideline we should avoid `protected` access modifier. @kushagraThapar please comment if protected is fine here.

Reply:
JsonSerializable is an implementation type now. These methods could be public too :)